### PR TITLE
[IMP] subscriptions: update subscription plans

### PR DESCRIPTION
**Pull request by meng-odoo**
Update to the emergency subscriptions update for Odoo 16 (eliminating subscription templates from Odoo 15).
Related to task #[3222232](https://www.odoo.com/web#id=3222232&menu_id=4720&cids=3&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/sales/subscriptions/plans.rst [hunk 1]
```diff
@@ -2,45 +2,96 @@
...
-   On each template, you can add specific :doc:`Terms & Conditions
-   <../sales/send_quotations/terms_and_conditions>`. If **terms and conditions** are specified on a
-   template, these will be used instead of the default **terms and conditions** set in the **Sales**
-   app settings (if any are set).
+   :align: center
+   :alt: Subscription plan (quotation template) configuration form.
+
+The subscription plan form contains the following options:
+
+- The name of the plan: Enter a name for the subscription plan at the top of the page.
```
Comment (hojo-odoo) [inline content/applications/sales/subscriptions/plans.rst:24]:
```suggestion
- Title: Enter a name for the subscription plan at the top of the page.
```

File: content/applications/sales/subscriptions/plans.rst [hunk 2]
```diff
@@ -2,45 +2,96 @@
...
+- :guilabel:`Quotation expires after`: Enter the number of days after which the quotation expires,
+  starting from the day the quotation is sent to the customer. Leave this field at zero for the
+  quotation to never expire.
+- :guilabel:`Online Confirmation`: Check the boxes next to :guilabel:`Signature` or
+  :guilabel:`Payment` to enable the customer to confirm their subscription order by signing or
+  paying for the quotation. Enable both to leave the choice to the customer. Enable neither to only
```
Comment (hojo-odoo) [inline content/applications/sales/subscriptions/plans.rst:30]:
```suggestion
  paying for the quotation. Enable both to leave the choice up to the customer. Enable neither to only
```

File: content/applications/sales/subscriptions/plans.rst [hunk 3]
```diff
@@ -2,45 +2,96 @@
...
+  - To create a new email template, enter a name for the template, then click :guilabel:`Create and
+    edit`.
+  - To edit an existing email template, select one from the drop-down menu, then click on the
+    :guilabel:`Internal link` arrow at the end of the line.
+
+- :guilabel:`Recurrence`: Select the recurrence period used for the plan. The recurrence periods
+  available here are the same ones that are configured in :menuselection:`Subscriptions -->
+  Configuration --> Recurrence Periods`.
+
+Selecting a :guilabel:`Recurrence` turns the quotation template into a subscription plan and enables
```
Comment (hojo-odoo) [inline content/applications/sales/subscriptions/plans.rst:46]:
```suggestion
Selecting a :guilabel:`Recurrence` period turns the quotation template into a subscription plan and enables
```

File: content/applications/sales/subscriptions/plans.rst [hunk 4]
```diff
@@ -2,45 +2,96 @@
...
+From the *Subscriptions* dashboard, click :guilabel:`New` to create a new quotation. Then, select a
+subscription plan in the :guilabel:`Subscription Plan` field.
+
+The :guilabel:`Recurrence`, products, and other information from the plan are automatically filled
+in. The quotation can then be modified further as needed.
+
+From the *Sales* dashboard, click :guilabel:`New` to create a new quotation. Then, select a
+subscription plan in the :guilabel:`Quotation Template` field.
+
+All subscription orders will appear on the *Subscriptions* dashboard regardless of whether they were
```
Comment (hojo-odoo) [inline content/applications/sales/subscriptions/plans.rst:96]: Same as above.

**Review by hojo-odoo — CHANGES_REQUESTED** [review]
Hey @meng-odoo,

Great job on this! Most of my comments are pretty minor. Make sure to decapitalize all of the words that follow a colon.

Thanks,
hojo

**Comment by meng-odoo** [comment]
Thanks for your review @hojo-odoo! FYI I followed all of the colons with a complete sentence so that they could properly be in sentence case (have a capitalized first letter).

Hi @samueljlieber, this is ready for technical review.

File: content/applications/sales/subscriptions/plans.rst [hunk 5]
```diff
@@ -2,45 +2,97 @@
...
+Selecting a :guilabel:`Recurrence` turns the quotation template into a subscription plan and enables
+the following additional options:
+
+- :guilabel:`Duration`: Choose whether the subscription plan has no end date (:guilabel:`Forever`)
+  or a :guilabel:`Fixed` duration.
+
+  - If the duration is :guilabel:`Forever`, then the subscription plan will continually renew until
+    either the customer or the company manually ends the subscription.
+  - If the duration is :guilabel:`Fixed`, then enter an :guilabel:`End After` date, which
+    determines the amount of time after which the subscription will automatically end.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/plans.rst:55]: early 100th char line break

File: content/applications/sales/subscriptions/plans.rst [hunk 6]
```diff
@@ -2,45 +2,97 @@
...
+
+- :guilabel:`Self Closable`: Check this box to enable the customer to terminate their subscription
+  from the :doc:`customer portal
+  </applications/websites/ecommerce/ecommerce_management/customer_accounts>`.
+- :guilabel:`Automatic Closing`: Enter the number of days after which *unpaid* subscriptions
+  *past* the due date are automatically closed.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/plans.rst:61]: early 100th char line break

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @meng-odoo 👋 I made just a couple of technical suggestion on this PR, please review them below and let me know if you have any questions. 

Note on images: Since were improving this doc, can you please resize the images used in `plans.rst` to be a maximum of 933px wide, ideally 768px wide. 

Thank you! 🙏

**Comment by meng-odoo** [comment]
Thank you @samueljlieber! I incorporated your edits (and added an image while I was at it) in the latest commit. Requesting your review again just in case :)

**Review by samueljlieber — APPROVED** [review]
Hi @meng-odoo! This PR is good to go 🚀 great job! @StraubCreative sending this to you for final review!

**Pull request closed without merging**